openmetadata-docs/content/v1.6.x/deployment/security/ldap/kubernetes.md

```diff
@@ -0,0 +1,109 @@
+
+## Authentication Configuration
+
+Update the `openmetadata.yaml` file with the following settings to enable LDAP authentication:
+
+```yaml
+global:
+  authentication:
+    provider: ldap
+    enableSelfSignup: false
+    publicKeyUrls:
+      - "https://<your-domain>/api/v1/system/config/jwks" # Replace with your domain
+    authority: "https://<your-domain>" # Replace with your domain
+    ldapConfiguration:
+      host: "ldap.example.com"  # Replace with your LDAP server hostname
+      port: 636  # Use 636 for secure LDAP (LDAPS) or 389 for standard LDAP
+      dnAdminPrincipal: "cn=admin,dc=example,dc=com"  # LDAP admin user DN
+      dnAdminPassword: "your-secret-password"  # Password for the admin user
+      userBaseDN: "ou=users,dc=example,dc=com"  # Base DN for LDAP users
+      mailAttributeName: "mail"  # Attribute for email in the LDAP schema
+      sslEnabled: true  # Enable SSL for secure LDAP
+      truststoreConfigType: "TrustAll"  # Trust store type (options: TrustAll, JVMDefault, HostName, CustomTrustStore)
+      trustStoreConfig:
+        trustAllConfig:
+          examineValidityDates: true  # Reject certificates outside the validity window
+  authorizer:
+    adminPrincipals: ["admin"]  # Add admin users here
+    principalDomain: "example.com"  # Organization domain for principal matching
+openmetadata:
+  server:
+    configOverrides:
+      security:
+        authProvider: ldap
+        jwtToken:
+          enabled: true  # Enable JWT tokens for secure communication
+          publicKeyUrls:
+            - "https://<your-domain>/api/v1/system/config/jwks"  # Replace with your domain
+
+## Mandatory Fields for LDAP Configuration
+
+- **provider**: Set to `ldap` for enabling LDAP authentication.
+- **publicKeyUrls**: Provide the public key URL in the format `{http|https}://{your_domain}:{port}/api/v1/system/config/jwks`.
+- **authority**: Specify your domain (e.g., `your_domain`).
+- **enableSelfSignup**: Set to `false` for LDAP.
+
+## Key LDAP Fields
+
+- **host**: Hostname of the LDAP server (e.g., `localhost`).
+- **port**: Port of the LDAP server (e.g., `10636`).
+- **dnAdminPrincipal**: The Distinguished Name (DN) of the admin principal (e.g., `cn=admin,dc=example,dc=com`).
+- **dnAdminPassword**: Password for the admin principal.
+- **userBaseDN**: Base DN for user lookups (e.g., `ou=people,dc=example,dc=com`).
+
+## Optional Advanced Configuration
+
+- **maxPoolSize**: Maximum connection pool size.
+- **sslEnabled**: Set to `true` to enable SSL connections to the LDAP server.
+- **truststoreConfigType**: Determines the type of trust store to use (`CustomTrustStore`, `HostName`, `JVMDefault`, or `TrustAll`).
+
+## Example: TrustStore Configurations
+
+### TrustAll Configuration
+
+```truststoreConfigType: TrustAll
+trustStoreConfig:
+  trustAllConfig:
+    examineValidityDates: true
+
```

Comment:
Hi, This is missing the closing backticks (3 backticks) for Code Block. Please update this.